RELEASE_NOTES.md

```diff
@@ -0,0 +1,72 @@
+# Performance Optimizations
+## Intel Architecture Processors
+* Improved matmul and inner product primitives performance on Intel Xeon processors with support for the Intel AMX instruction set.
+* Improved performance of convolution and inner product primitives on processors with Intel AVX2 instruction set support.
+* Improved performance of `int8` convolution support with zero points.
+* Improved `fp32` convolution performance with `fp16` and `bf16` compressed weights on processors with Intel AVX2 and AVX512 instruction set support.
+* Improved `fp16`/`bf16` depthwise convolution performance with `fp32` bias or `sum` post-ops or dilation.
+* Improved `bf16` pooling backpropagation performance.
+* Improved binary post-ops performance with `per_w` broadcast.
+
+## Intel Graphics Products
+* Improved performance on Intel GPUs based on Xe3 architecture.
+* Improved convolution performance on:
+ * Intel Arc Graphics for Intel Core Ultra (Series 2, formerly Lunar Lake).
+ * Intel Arc B-series discrete graphics (formerly Battlemage).
+* Improved `int8` matmul performance with zero-points support for source and weight tensors.
+* Improved matmul and reorder performance for 4-bit floating-point data types `f4_e2m1` and `f4_e3m0`. Compute primitives provide support through internal converison into f16 as current Intel GPUs lack native support.
+* Improved performance of the following subgraphs with Graph API:
+ * Scaled Dot Product Attention (SDPA) with `int4` and `int8` KV cache.
+ * SDPA with bottom-right implicit causal mask.
```

Comment:
We decided to not backport the bottom-right causal mask to rls-v3.8. Let remove it and add below:

```
* Differentiated SDPA with f32 or bf16/f16 intermediate data types. Now only SDPA with f32 intermediate data type will dispatch to the optimized kernels on Intel Graphics Products.
```

Reply:
Done.